src/irc/IrcServer.ts

```diff
@@ -34,6 +34,9 @@ export interface IrcServerConfig {
     ssl?: boolean;
     sslselfsign?: boolean;
     sasl?: boolean;
+    tlsOptions?: {
+
+    };
```

Comment:
Is there anything missing?
```suggestion
    tlsOptions?: {};
```